modules/iframe-app/local.config-node.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
This should go with our other default config files in `ops/config`

Reply:
You're right about this. The whole config strategy has changed since now this is all in the iframe. The config for the iframe just needs to be the default node config now. Fixed this to use the same config.